### PR TITLE
Eugene - add different button styles

### PR DESCRIPTION
**Pull request by yevhensydorov**

### CODE REVIEW

**Review by 40thieves — COMMENTED** [review]
This looks great! 🎉 👍 🙂 

The next steps are to try adding a `<ul>` and then try using a Sass loop to style 3 paragraphs.

**Comment by yevhensydorov** [comment]
@40thieves 
Just added two ul's with different styles

File: skin.css [hunk 1]
```diff
@@ -3,11 +3,91 @@
...
+	padding: 10px 60px;
   border-radius: 3px;
-  border-style: solid;
-  border-width: 1px;
+  border: 1px #CECECE solid;
+  background: #fff;
   font-weight: normal;
   line-height: 1.15;
   text-align: center;
   vertical-align: middle;
```
Comment (OlenaKashuba) [inline skin.css:13]: vertical-align and text-align are useless for block elements such as button

**Review by 40thieves — COMMENTED** [review]
Looks great 🎉 Nice work!

Could you fix up @OlenaKashuba's comment?

**Comment by yevhensydorov** [comment]
@40thieves done

**Pull request closed without merging**